i18n/language/language.go

```diff
@@ -17,23 +19,35 @@ type Language struct {
 	*PluralSpec
 }
 
+// NewLanguage returns new Language with specified tag and spec.
+func NewLanguage(tag string, spec *PluralSpec) *Language {
```

Comment:
I made it only because it's a good tone that you should use a constructor for creating objects, not using the `&Language{}` annotation.